crates/bevy_text/src/text.rs

```diff
@@ -510,7 +508,7 @@ pub fn detect_text_needs_rerender<Root: Component>(
             ));
             continue;
         };
-        let mut parent: Entity = **span_parent;
+        let mut parent: Entity = span_parent.0;
```

Comment:
If we impl `Deref` for `Parent` we should be able to revert these changes and reduce breakage in the engine and for users.

Reply:
I personally think `**` is _significantly_ nastier and would prefer to remove it. Reducing breakage is nice, but im not sure deferring this pain (if we agree that it is inevitable) will be particularly beneficial.

Reply:
Given that `Entity` doesn't have any meaningful methods on it, I think that's a reasonable position to take.

Reply:
Although I guess another reasonable position to take is that a Deref impl doesn't really hurt anyone and we can always just not use it by convention. Nevermind I'll add it in the interest of keeping breakage down, and just stop using it in engine code.